components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/oauth2/token/AccessTokenIssuerTest.java

```diff
@@ -1,770 +1,300 @@
-//                context.setScope(scopeArray);
-//                return new OAuth2AccessTokenRespDTO();
-//            }
-//        });
-//
-//        HashMap<String, AuthorizationGrantHandler> authorizationGrantHandlers = new HashMap<>();
-//        authorizationGrantHandlers.put(grantType, grantHandler);
-//
-//        mockOAuth2ServerConfiguration(authorizationGrantHandlers);
-//    }
-//
-//    private IDTokenBuilder getMockIDTokenBuilderForSuccess() throws IdentityOAuth2Exception {
-//
-//        IDTokenBuilder idTokenBuilder = mock(IDTokenBuilder.class);
-//        when(idTokenBuilder.buildIDToken(any(OAuthTokenReqMessageContext.class), any(OAuth2AccessTokenRespDTO.class)))
-//                .then(new Answer<Object>() {
-//                    @Override
-//                    public Object answer(InvocationOnMock invocationOnMock) throws Throwable {
-//
-//                        return ID_TOKEN;
-//                    }
-//                });
-//        return idTokenBuilder;
-//    }
-//
-//    private IDTokenBuilder getMockIDTokenBuilderForFailure() throws IdentityOAuth2Exception {
-//
-//        IDTokenBuilder idTokenBuilder = mock(IDTokenBuilder.class);
-//        when(idTokenBuilder.buildIDToken(any(OAuthTokenReqMessageContext.class), any(OAuth2AccessTokenRespDTO.class)))
-//                .thenThrow(new IDTokenValidationFailureException("ID Token Validation failed"));
-//        return idTokenBuilder;
-//    }
-//
-//    private void mockPasswordGrantHandler(boolean isAuthorizedClient, boolean isValidGrant,
-//                                          boolean isAuthorizedAccessDelegation, boolean isValidScope)
-//            throws IdentityOAuth2Exception {
-//
-//        Map<String, AuthorizationGrantHandler> authzGrantHandlers = new Hashtable<>();
-//
-//        when(passwordGrantHandler.isOfTypeApplicationUser()).thenReturn(true);
-//        when(passwordGrantHandler.isAuthorizedClient(any(OAuthTokenReqMessageContext.class)))
-//                .thenReturn(isAuthorizedClient);
-//        when(passwordGrantHandler.validateGrant(any(OAuthTokenReqMessageContext.class))).thenReturn(isValidGrant);
-//        when(passwordGrantHandler.authorizeAccessDelegation(any(OAuthTokenReqMessageContext.class)))
-//                .thenReturn(isAuthorizedAccessDelegation);
-//        when(passwordGrantHandler.validateScope(any(OAuthTokenReqMessageContext.class))).thenReturn(isValidScope);
-//        when(passwordGrantHandler.issue(any(OAuthTokenReqMessageContext.class)))
-//                .thenReturn(new OAuth2AccessTokenRespDTO());
-//        authzGrantHandlers.put("password", passwordGrantHandler);
-//        when(passwordGrantHandler.isConfidentialClient()).thenReturn(true);
-//
-//        when(oAuthServerConfiguration.getSupportedGrantTypes()).thenReturn(authzGrantHandlers);
-//    }
+    private OAuthAppDO appDO;
+
+    private final String jwt = "eyJ4NXQiOiJ4WFJRdkZUOFNtLUpQUEFrY0loNHlUTlhKTkkiLCJraWQiOiJNREExWXpKbU0yWmxZV1E1TldN"
```

Comment:
rename to testJwt or a suitable name

Reply:
Addressed with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2815/commits/0f8bba9c04b7dd14538fe4937ff5d7825c3f3402